<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgXyView</class>
 <widget class="QDialog" name="dlgXyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Shot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_7">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbxShotNumX">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cbxChannelX"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_9">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbxShotNumY">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cbxChannelY"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnPlot">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
